app/assets/styles/partials/components/_question.scss

```diff
@@ -38,6 +39,14 @@
   }
 }
 
+.question__title__number {
+  padding-right: 0.5rem;
+  @include mq(xl) {
+    position: absolute;
+    left: -48px;
```

Comment:
If you use `text-align: right` and then give this a fixed width then it'll scale better with bigger numbers:

![screenshot 2017-02-01 14 08 16](https://cloud.githubusercontent.com/assets/930398/22509928/f3423eb0-e887-11e6-8a6a-3470c48f95ce.png)

![screenshot 2017-02-01 14 08 37](https://cloud.githubusercontent.com/assets/930398/22509935/fa33f6d2-e887-11e6-984f-c7570ef26336.png)